Code/Tools/ProjectManager/Source/AddRemoteProjectDialog.cpp

```diff
@@ -162,14 +169,56 @@ namespace O3DE::ProjectManager
         // Simulate repo being entered and UI enabling
```

Comment:
this comment no longer seems valid - isn't this code now used to validate the URI a second after the user changes the text?